packages/ra-core/src/controller/field/useReferenceField.ts

```diff
@@ -64,26 +61,22 @@ export interface UseReferenceProps {
  *
  * @returns {ReferenceProps} The reference props
  */
-export const useReference = ({
+export const useReferenceField = ({
```

Comment:
As far as I understand, this hook just differs from the `useReference` hook by returning the link target. As this extra logic does not involve React hooks, you should extract it to a pure function, and remove the `useReferenceField` hook to leave just the `useReference` hook.